doc/whats_new/v0.22.rst

```diff
@@ -495,6 +496,16 @@ Changelog
   requires less memory.
   :pr:`14108`, :pr:`14170`, :pr:`14296` by :user:`Alex Henrie <alexhenrie>`.
 
+- |Efficiency| :class:`linear_model.RidgeCV` now does not allocate a
+  potentially large array to store dual coefficients for all hyperparameters
+  during its `fit`, nor an array to store all LOO predictions unless
```

Comment:
LOO predictions or mean squared errors

Reply:
actually it depends of `scoring`: LOO predictions if `scoring is not None` otherwise mean squared errors.
But agreed that it should be added.